runelite-client/src/main/java/net/runelite/client/util/ImageUtil.java

```diff
@@ -382,10 +383,12 @@ public static BufferedImage getResourceStreamFromClass(final Class c, final Stri
 		}
 		catch (IllegalArgumentException e)
 		{
+			log.error("Failed to load image from class: {}, path: {}", c.getName(), path);
```

Comment:
Wouldn't this just log something like `Failed to load image from class: ExamplePlugin, path example.png`? That would not be helpful since it would not describe the physical file path where the attempted load would have occurred.